cmd/agent/subcommands/remoteconfig/command.go

```diff
@@ -64,7 +75,7 @@ func state(cmd *cobra.Command, args []string, dialOpts ...grpc.DialOption) error
 	}
 	ctx = metadata.NewOutgoingContext(ctx, md)
 
-	cli, err := agentgrpc.GetDDAgentSecureClient(ctx, dialOpts...)
+	cli, err := agentgrpc.GetDDAgentSecureClient(ctx)
```

Comment:
Why this change?

Reply:
This slice is always empty (`state` is only called in one place), so I just removed it rather than build a way to pass it with Fx.